Vagrantfile

```diff
@@ -2,7 +2,8 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-  config.vm.box = "chef/centos-6.5"
+  config.vm.box = "chef/centos-6.6"
+  #config.vm.box = "bhosmer/centos6.6-minimal"
```

Comment:
does it work with bhosmer/centos6.6-minimal?  Is the minimal vm better (ie. faster setup)? 

I'm just trying to figure if this an option generally applicable for others or should we just drop this commented line?